### PR TITLE
Bug fix logging error strings

### PR DESCRIPTION
**Pull request by rifkinni**
# Bug
We were not including error fields when `logger.error` was called with both a string message and an error

# Fix
We were failing to reassign the merged content of the error fields back to the original message variable https://github.com/simplybusiness/twiglet-ruby/commit/3d74c881c5d8f560d26545d06e2eb9d1a5b7ffa9

# Tech notes
This is an alternate solution to https://github.com/simplybusiness/twiglet-ruby/pull/10

### CODE REVIEW

**Review by addersuk — APPROVED** [review]

**Review by punitcse — APPROVED** [review]
🍰

**Review by pawelma — APPROVED** [review]
🎉

File: README.md [hunk 1]
```diff
@@ -42,7 +42,8 @@ This will write to STDOUT a JSON string:
 
 Obviously the timestamp will be different.
 
-Alternatively, if you just want to log some error message in text format
+Alternatively, if you just want to log some error string:
+
 ```ruby
 logger.error( "Emergency! There's an Emergency going on")
```
Comment (pawelma) [inline README.md]: unnecessary whitespace before `"Emrgency...`
